### PR TITLE
Update devnook.md

### PR DESCRIPTION
**Pull request by devnook**
Description 234567

### CODE REVIEW

**Comment by jpmedley** [comment]
Do you need me to do something with this?

File: site/en/devnook.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 permalink: "/devnook.txt"
 ---
-Devnook test 2
```
Comment (tomayac) [inline site/en/devnook.md:4]: This should be 5.

**Review by tomayac — CHANGES_REQUESTED** [review]
Getting there.

**Review by jeffposnick — APPROVED** [review]
Tom doesn't know what he's talking about. This PR is ace.

**Comment by devnook** [comment]
comment

**Comment by devnook** [comment]
comment

**Pull request closed without merging**